.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
add yarn-debug.log
Update the file

You can get inspired from the github generated file ([Example](https://github.com/xcomponent/react-gojs/blob/master/.gitignore)

Reply:
done